### PR TITLE
Development for changes to LIBRS - JSON Specifications, API Validator: Source code for API endpoints, JSON model changes, and updates to docs.librs.org specifications ready for review in TEI test environment

### PR DESCRIPTION
**Pull request by john-maus**
# Changes:
- Corrected documentation for how age is entered and handled.

# Asana Tasks Involved:
- [Development for changes to LIBRS - JSON Specifications, API Validator: Source code for API endpoints, JSON model changes, and updates to docs.librs.org specifications ready for review in TEI test environment](https://app.asana.com/0/1207380286165003/1207380288238781)
- [Offender Object - spec update](https://app.asana.com/0/1207380286165003/1207798733022211)
- [Victim Object - Data type changes for string and spec update](https://app.asana.com/0/1207380286165003/1207798733022212)
- [Arrestee Object - Spec update](https://app.asana.com/0/1207380286165003/1207798733022213)
- [Development for changes to LIBRS - updates to docs.librs.org specifications](https://app.asana.com/0/1207380286165003/1207380288238779)
- [Disposition of Arrestee Under 18 (Spec wording changes only)](https://app.asana.com/0/1207380286165003/1207798727914757)

# Testing:
- Verify that the offender age and estimatedAge documentation matches what Kate specified in the Asana task. [PASSED]
- Verify that the victim age and estimatedAge documentation matches what Kate specified in the Asana task. [PASSED]
- Verify that the arrestee age, estimatedAge, and dispositionUnder18 documentation matches what Kate specified in the attachment in the Asana task. [PASSED]
- Verify that the JSON examples for offender, victim, and arrestee are updated to reflect documentation changes. [PASSED]

# Notes:
- I'm not sure why, but VS Code made some whitespace changes to the markdown file I edited since I opened it in a markdown editing extension within VS Code. @dougbarbin - You may have to scroll past the text in the changes that looks repeated and just worry about the changes for offender, victim, and arrestee. See the Asana tasks involved above for the changes that needed to be made.
- Also, I forgot to make a new branch for the other document changes I just made, so it got included in this PR and I am updating the description.

### CODE REVIEW

File: _data/data-element-defs.json [hunk 1]
```diff
@@ -2319,7 +2319,7 @@
 	}, {
 		"de_num": "52",
 		"name": "DISPOSITION OF ARRESTEE UNDER 18",
-		"definition": "Used only if the arrestee was 16 years of age or younger at the time of arrest.",
+		"definition": "Mandatory field for ANY Arrestee that is 17 years of age and younger. The Federal UCR program considers juveniles as under 18 years of age regardless of state law. The LIBRS program will stay aligned with the FBI and collect this field for arrestees 17 and younger even though this differs from the latest state Juvenile Age Definition.\n\nIf the arrestee is 17 and considered an adult in Louisiana after April 19, 2024, please use A – Referred to Criminal or Adult Court.",
```
Comment (dougbarbin) [inline _data/data-element-defs.json]: @john-maus please re-word this. An Arrestee cannot possibly be 17 years of age AND younger; they can only be one or the other. It should read, "...ANY Arrestee that is 17 years of age or younger."

**Comment by dougbarbin** [comment]
@john-maus please see my comment about the slight verbiage change. Other than that, this looks good to go.

**Comment by dougbarbin** [comment]
> @john-maus please see my comment about the slight verbiage change. Other than that, this looks good to go.

@kirkes21 - I asked Kate about it. There might be some specific verbiage we have to use. It's a little confusing and the context matters - are we talking about the Arrestee or their age, for example? Here's a different diff (see what I did there) where it explicitly mentions Age, and in that context, "and" definitely makes sense. Plurality might also make a difference (e.g., Arrestees versus Arrestee). Anyways, let's see what Kate says. 

image

**Comment by john-maus** [comment]
> > @john-maus please see my comment about the slight verbiage change. Other than that, this looks good to go.
> 
> @kirkes21 - I asked Kate about it. There might be some specific verbiage we have to use. It's a little confusing and the context matters - are we talking about the Arrestee or their age, for example? Here's a different diff (see what I did there) where it explicitly mentions Age, and in that context, "and" definitely makes sense. Plurality might also make a difference (e.g., Arrestees versus Arrestee). Anyways, let's see what Kate says.
> 
> image

@dougbarbin I just copied how Kate had it in the word document from the task. I think we should ask her first?

File: _data/data-element-defs.json (hunk 1 above)
Comment (john-maus) [reply _data/data-element-defs.json]: This is done.

**Comment by dougbarbin** [comment]
@john-maus - this broke the https://docs.librs.org site, so I had to revert it.

**Pull request merged**